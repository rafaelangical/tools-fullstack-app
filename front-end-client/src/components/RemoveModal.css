.App-modal-remove{
  width: 100%;
  height: 100%;
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  align-items: center;
  position: absolute;
  position: fixed;
  background-color: rgba(1,1,1,0.5); 
  top: 0;
}
.div-modal-remove-full {
  width: 55%;
  height: 20%;
  margin: 0 auto;
  background-color: white;
  display: flex;
  flex-flow: row wrap;
  border: 1px solid black;
}
.div-modal-remove-top{
  width: 90%;
  height: 60%;
  margin: 0 auto;
  display: flex;
  flex-flow: column wrap;
  justify-content: space-between;
}
.div-modal-remove-bottom{
  width: 100%;
  height: 30%;
  margin: 0 auto;
  display: flex;
  justify-content: flex-end;
}
.div-modal-remove-bottom > button{
  display: block;
  margin-right: 5%;
  width: 30%; 
  height: 70%;
  border-radius: 1em;
}
.remove-tool-button{
  border: 1px solid red;
}