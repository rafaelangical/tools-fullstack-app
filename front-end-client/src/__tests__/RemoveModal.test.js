import React from 'react';
import RemoveModal from '../components/RemoveModal';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() })

describe('Test if RemoveModal is rendered correctly', () => {
  test('renders correctly removeModal', () => {
    const tree = renderer.create(
      <RemoveModal />
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
