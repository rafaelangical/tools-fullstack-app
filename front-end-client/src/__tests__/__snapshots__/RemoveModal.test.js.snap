// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if RemoveModal is rendered correctly renders correctly removeModal 1`] = `null`;
